contracts/priv/cloud-priv.yml

```diff
@@ -619,6 +736,66 @@ components:
                     user:
                       readOnly: true
                       $ref: '#/components/schemas/Link'
+    OnboardingWithIDsRequest:
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/Identity'
+        org:
+          $ref: '#/components/schemas/Identity'
+        config:
+          type: object
+          properties:
+            bucket:
+              type: string
+              description: name of non system bucket
+            retentionPeriodSeconds:
+              type: integer
+              description: retention period of non system bucket
+            limit:
+              $ref: '#/components/schemas/Limit'
+        returnToken:
+          type: bool
+          description: if true will return the auth token in the response
```

Comment:
Is `auth token` the right description here? I'm not sure how tokens are named in the platform; this token should be an org-scoped token that allows the OEM customer to manage the new organization, but I'm not sure what the right name for that is.

Reply:
What do you want to call it.

Reply:
I looked at the API docs, and the endpoints for this are named around the term `authorization`, so this wording is fine. I think I've been working with the JWT/shared session token too much :-).

Reply:
I change it to token